manifest.yaml

```diff
@@ -5,6 +5,8 @@ modules:
     version: ">=2.33.0"
 stencilVersion: "^v1.43.0-rc.5"
 postRunCommand:
+  - name: Trust mise config
```

Comment:
fwiw this is built into the [new stencil](https://github.com/rgst-io/stencil/blob/be121151b71f52d68da4ec738d0f4e8a7b2410dd/internal/codegen/stencil.go#L293) 😛